pyhap/accessory_driver.py

```diff
@@ -519,6 +491,15 @@ def start(self):
         if not self.accessory.paired:
             self.accessory.setup_message()
 
+        # Start the accessory so it can do stuff.
+        self.accessory.set_sentinel(self.stop_event, self.aio_stop_event, self.event_loop)
+        self.accessory_task = self.event_loop.create_task(self.accessory.start())
+        logger.info("Starting event loop")
+        self.event_loop.run_until_complete(self.accessory_task)
+        self.event_loop.stop()
```

Comment:
No need to call `stop`. `run_until_complete` already does this for you. (In fact, `run_until_complete` can only return once the loop has stopped)

Reply:
Done